acceptance/windows-service/test/integration/chef-windows-service/inspec/chef_windows_service_spec.rb

```diff
@@ -0,0 +1,58 @@
+only_if do
```

Comment:
I believe it is inspec only - I don't recall seeing it in rspec.
See https://github.com/chef/inspec/tree/master/docs for usage in example.